src/docs/asciidoc/security.adoc

```diff
@@ -218,94 +219,124 @@ Hazelcast provides a default SSLContextFactory, `com.hazelcast.nio.ssl.BasicSSLC
 ```
 
 Here are the descriptions for the properties:
- 
-* `keystore`: Path of your keystore file. Note that your keystore's type must be `JKS`.
+
+* `keystore`: Path of your keystore file.
 * `keyStorePassword`: Password to access the key from your keystore file.
 * `keyManagerAlgorithm`: Name of the algorithm based on which the authentication keys are provided.
-* `keyStoreType`: The type of the keystore. Its default value is `JKS`. 
-* `truststore`: Path of your truststore file. The file truststore is a keystore file that contains a collection of certificates trusted by your application. Its type should be `JKS`.
+* `keyStoreType`: The type of the keystore. Its default value is `JKS`.
+* `truststore`: Path of your truststore file. The file truststore is a keystore file that contains a collection of certificates trusted by your application.
 * `trustStorePassword`: Password to unlock the truststore file.
 * `trustManagerAlgorithm`: Name of the algorithm based on which the trust managers are provided.
-* `trustStoreType`: The type of the truststore. Its default value is `JKS`. 
+* `trustStoreType`: The type of the truststore. Its default value is `JKS`.
+* `mutualAuthentication`: Mutual authentication configuration. It's empty by default which means the client side of connection is not authenticated. Available values are:
+** `REQUIRED` - server forces usage of a trusted client certificate
+** `OPTIONAL` - server asks for a client certificate, but it doesn't require it
++
+Please see the <<mutual-authentication, Mutual Authentication section>>.
+* `ciphersuites`: Comma-separated list of cipher suite names allowed to be used. Its default value are all supported suites in your Java runtime.
```

Comment:
Maybe including this property (with an example value) in the above <ssl> declarative configuration is a good idea?

Reply:
Actually there is kind of one: http://docs.hazelcast.org/docs/latest-dev/manual/html-single/index.html#configuring-cipher-suites

But you are right, out of scope here.